app/models/split.rb

```diff
@@ -6,6 +6,13 @@ class Split < ActiveRecord::Base
   has_many :bulk_assignments, dependent: :nullify
   has_many :variant_details, dependent: :nullify
 
+  has_many :remote_kills, ->(split) { where(split: split) }, through: :owner_app, class_name: 'AppRemoteKill', inverse_of: :split
```

Comment:
i don't follow this relation. why use `through: :owner_app`? remote_kill has a foreign key to split